src/Pulsar.Client/Api/ITableView.fs

```diff
@@ -0,0 +1,28 @@
+namespace Pulsar.Client.Api
+
+open System
+open System.Collections.Generic
+
+type ITableView<'T> =
```

Comment:
I've just got an idea - insted of making all those custom abstrace members you can just inherit from IReadOnlyDictionary interface and implement it's methods

Reply:
The `Keys` and `Values` should be methods, but in `IReadOnlyDictionary`, it is a property, I don't think use the `IReadOnlyDictionary` can implement this.

Reply:
Right, I think they should be properties rather than methods :) The underlying ConcurrentDictionary has them as properties as well. In .NET properties are widely used for such things instead of methods

Reply:
Thanks,  please review.